base/src/test/java/io/fabric8/launcher/base/PathsTest.java

```diff
@@ -55,5 +55,15 @@ public void unzip() throws IOException {
         assertThat(tempDirectory).exists();
     }
 
+    /*
+     * See https://github.com/fabric8-launcher/launcher-backend/pull/541
+     */
+    @Test
+    public void unzip_zip_without_dir_entries() throws IOException {
+        Path tempDirectory = temporaryFolder.newFolder().toPath();
+        Files.delete(tempDirectory);
+        Paths.unzip(new FileInputStream("src/test/resources/io/fabric8/launcher/base/pathtest.zip"), tempDirectory);
```

Comment:
`try (InputStream is = getClass().getResourceAsStream("pathtest.zip")){
...
}`

Reply:
`assertThatCode(()->Paths.unzip).doesNotThrowAnyException();

Reply:
Ok done, but isn't the assert implicit in the test?